proposals/0284-DialNumber_Text.md

```diff
@@ -28,19 +28,46 @@ Apps can use a different message based on the use case:
 Add a text field to DialNumber in the MOBILE_API and HMI_API. 
 The HMI would still need to display the phone number and a call button so that the user understands that a call is about to be placed. If the app doesn't send a message, the HMI would just display the standard string that they currently do for `DialNumber`.
 
+Add to the `MOBILE_API`:
 ```xml
     <function name="DialNumber" functionID="DialNumberID" messagetype="request" since="3.0">
     .
     .
     .
-        <param name="messageText" type="Common.TextFieldStruct" mandatory="false" since="X.X">
+        <param name="promptText" type="Common.TextFieldStruct" mandatory="false" since="X.X">
             <description>
               Body of text to display to the user.
             </description>
         </param>
     </function>

```

Comment:
```suggestion
    <param name="promptText" type="Common.TextFieldStruct" mandatory="false" since="X.X">
        <description>Body of text to display to the user.</description>
    </param>
</function>
```